includes/pull-ui.php

```diff
@@ -76,6 +76,13 @@ function setup_list_table() {
 		}
 	}
 
+	// Current user roles.
+	$current_user_roles = (array) wp_get_current_user()->roles;
+
+	// Check if user has cap to pull any external connection.
+	$pull_capabilities        = apply_filters( 'dt_pull_capabilities', 'manage_options' );
```

Comment:
We should move the filter docblock to above this, so this gets added properly to our doc site